shell/agents/AIShell.Azure.Agent/Telemetry/MetricHelper.cs

```diff
@@ -41,7 +41,7 @@ private void InitializeTelemetryClient()
         // Hence connection string must be specified here.
         services.AddApplicationInsightsTelemetryWorkerService((ApplicationInsightsServiceOptions options) =>
             {
-                options.ConnectionString = "InstrumentationKey=c7d054ff-9f40-43e8-bf8e-7d76c58cc1af";
+                options.ConnectionString = "InstrumentationKey=eea660a1-d969-44f8-abe4-96666e7fb159";
```

Comment:
Can we add a comment to indicate which application insights we use here as GUID is not straightforward for understanding.

Reply:
Good idea, I added a comment for that but didn't add the resource name since this code is public.